scripts/coverage_report.sh

```diff
@@ -5,20 +5,29 @@
 # Check that the test directory and report path arguments are provided
 if [ $# -lt 2 ] || ! [ -d "$1" ]
 then
-        echo "Usage: $0 <testdir> <outdir> [--batch]"
+        echo "Usage: $0 <testdir> <outdir> [--batch [ --failed_crate_file <path> ]]"
```

Comment:
Doeds --failed_crate_file depend on --batch? If not, maybe write it `[--batch] [--failed_crate_file <path>]` instead. Also, @bmwill will want dashes instead of underscores :)

Reply:
yeah, unless you're in batch mode, you can't use the --failed_crate_file.

This is more a result of us not using getops in this shell initially and not wanting to break the existing api, and we only need the --failed_crate_file <path> in batch mode.

I'll make a change from the underscores to dashes.  I hope to pull this all in to x at some point soon however though.